tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py{27,36,37,38,39}
```

Comment:
We support python 3.5.

Reply:
Ah, sorry, I've included only the versions I was able to test locally.